README.md

```diff
@@ -7,39 +7,33 @@
 
 ## Samples
 
-ForgeRock provides these samples to help demonstrate SDK functionality/implementation. They are provided "as is" and are not official products of ForgeRock and are not officially supported.
+Ping provides these samples to help demonstrate SDK functionality/implementation. They are provided "as is" and are not official products of Ping and are not officially supported.
 
-To try out the ForgeRock JavaScript SDK please look at one of our samples:
+### Integrate with PingOne Advanced Identity Cloud / PingAM:
 
-- [**Embedded login - `embedded-login`**](./embedded-login/README.md)
+To try out the Ping JavaScript SDK please look at one of our samples under [javascript](./javascript/)
```

Comment:
I'm not sure about the naming in these three sentences, it's possibly also a bit wordy/repetitive.
Perhaps something like:
`We provide samples of integrating with Advanced Identity Cloud and PingAM for the following platforms:
 - Android
 - iOS
 - JavaScript`